spec/models/case_log_spec.rb

```diff
@@ -2235,4 +2235,24 @@ def previous_postcode_fields
       end
     end
   end
+
+  describe "csv download" do
+    let(:csv_export_file) { File.open("spec/fixtures/files/case_logs_download.csv", "r:UTF-8") }
+    let(:scheme) { FactoryBot.create(:scheme) }
+    let(:location) { FactoryBot.create(:location, scheme:, type_of_unit: 6, postcode: "SE11TE") }
+
+    before do
+      described_class.destroy_all
```

Comment:
Why is this necessary? Feels like a hack for a problem elsewhere?

Reply:
Hmm, just to make sure that we only have 1 record when we make the download. Doesn't seem to fail without it, so happy to remove